I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 125, in script
    jump refused
ScriptError: could not find label 'refused'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 125, in script
    jump refused
  File "D:\renpy-8.0.3-sdk\renpy\renpy\ast.py", line 1974, in execute
    rv = renpy.game.script.lookup(target)
  File "D:\renpy-8.0.3-sdk\renpy\renpy\script.py", line 927, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'refused'.

Windows-10-10.0.22000 AMD64
Ren'Py 8.0.3.22090809
Capybara 1.0
Sat Dec  3 14:07:30 2022
